README.md

```diff
@@ -120,6 +120,20 @@ There are two ways to contribute to Concrete:
 
 Only approved contributors can send pull requests, so please make sure to get in touch before you do!
 
+## Citing Concrete
+
+To cite Concrete in academic papers, please use the following entry:
+
+```
+@inproceedings{CJLOT20,
```

Comment:
I don't know if it's important, but we could follow the [cryptobib labeling conventions](https://cryptobib.di.ens.fr/manual) for the label. In that case the label would be `WAHC:CJLOT20`. The reasoning is that I've heard from at least a couple of researchers that they like the uniform format of the cryptobib labels, which makes them easier to find, memorize and collaborate. So this might make life a tad easier for a few people out there.